autometa/taxonomy/ncbi.py

```diff
@@ -237,23 +245,19 @@ def parent(self, taxid):
         int
             parent taxid if taxid is found in nodes.dmp or merged.dmp else 1
         """
-        self.is_valid_taxid(taxid)
-        parent_taxid = self.nodes.get(taxid, {'parent': 1}).get('parent')
-        if parent_taxid == {'parent': 1}:
-            return self.merged.get(taxid, {'parent': 1}).get('parent')
+        if not self.is_valid_taxid(taxid):
+            return False
```

Comment:
I'm noticing a trend here. Instead of copying and pasting this everywhere you need to consider the context of the method